containers/deploy/project.yml

```diff
@@ -1,7 +1,12 @@
 ---
+- set_fact:
+    project_name: "{{ project_name | default('foreman') }}"
+  tags:
+    - always
```

Comment:
I think putting this here makes the tasks in `foreman.yml` really order dependent. I think I would rather this be a task in `foreman.yml` than in one of the `deploy/*` task files.